src/Core/AspectKernel.php

```diff
@@ -254,7 +284,7 @@ protected function registerTransformers()
                 $transformers[] = new ConstructorExecutionTransformer();
             }
 
-            return $transformers;
+            return $aspectKernel->customizeTransformers($transformers);
```

Comment:
I was thinking of application code not taking care of preparing the "sandwich" of transformers correctly: application sets up linear transformations, and they are all wrapped up in cacher correctly by the framework.
I found it handy in my case. Anyway,  I'll remove the method, if it is crucial from your standpoint